lib/flutter_client_sse.dart

```diff
@@ -17,12 +17,10 @@ class SSEClient {
         request.headers["Cookie"] = token;
         Future<http.StreamedResponse> response = _client.send(request);
         await for (final data in response.asStream()) {
-          await for (final d in data.stream) {
-            final rawData = utf8.decode(d);
-            final event = rawData.split("\n")[1];
-            if (event != '') {
-              yield SSEModel.fromData(rawData);
-            }
+          final rawData = await data.stream.transform(utf8.decoder).join();
```

Comment:
@lakexyde adding this line seems to stop the execution to this and it keeps awaiting a response. Although using transform is the correct method to handle long responses, this stops the execution.

Reply:
@pratikbaid3  I have added a fix and handled reconnection. Thank you.